Cabal-syntax/src/Distribution/Types/LibraryName.hs

```diff
@@ -10,13 +10,15 @@ module Distribution.Types.LibraryName
   , libraryNameString
 
     -- * Pretty & Parse
+  , prettyLibraryNames
   , prettyLibraryNameComponent
   , parsecLibraryNameComponent
   ) where
 
 import Distribution.Compat.Prelude
 import Prelude ()
 
+import qualified Data.List.NonEmpty as NEL
```

Comment:
I don't really know the grouping followed in the cabal codebase but it feels like a block where all are `Distribution.` should not have `Data.List.NonEmpty` in it.